From ad393302a1e17889067fd139c28962b4b3791fc1 Mon Sep 17 00:00:00 2001
From: David Edmundson <kde@davidedmundson.co.uk>
Date: Mon, 17 Feb 2025 12:14:54 +0000
Subject: [PATCH] Fix systemd startup

plasma-workspace.target is explicitly marked as being before
graphical-session.target. If krdpserver is set to come up after
graphical-session.target this creates an issue that plasma-workspace
can't be marked as up until this is up, and we can't start
graphical-session yet.

Anything wanted by plasma-workspaec needs to have dependencies set
explicitly.

Ammends: df4d9d4ea68cccb35da0a5b55a32652c58b54acd

BUG: 500017
---
 server/app-org.kde.krdpserver.service.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/server/app-org.kde.krdpserver.service.in b/server/app-org.kde.krdpserver.service.in
index 4d0092e..36b91bf 100644
--- a/server/app-org.kde.krdpserver.service.in
+++ b/server/app-org.kde.krdpserver.service.in
@@ -2,7 +2,8 @@
 # SPDX-FileCopyrightText: 2024 Akseli Lahtinen <akselmo@akselmo.dev>
 [Unit]
 Description=KRDP Server
-After=graphical-session.target
+After=plasma-xdg-desktop-portal-kde.service
+After=plasma-core.target
 
 [Service]
 Type=exec
-- 
GitLab

